src/app/use-cases/request-signature.ts

```diff
@@ -62,6 +62,10 @@ export const makeRequestSignature =
           productId: payload.productId,
           signerId: signer.id,
           documents,
+          // WARNING! this is just a placeholder
+          // TODO: replace the static QR-code with a dynamic one
+          qrCodeUrl:
```

Comment:
can we put this into some constant named "mockSomething" (too have a recognizable pattern that clarifies this is a stub)? moreover I'd rather link something that's inside this repository.

Reply:
sure